get_matching_values/getmatchingvalue.js

```diff
@@ -0,0 +1,25 @@
+// receive 2 array. 
+// return matching value. if no match return null
+function getMatchingValue(a, b) {
+	
+	var result = [];
+	a.forEach(function (el) {
+		if (b.indexOf(el) != -1) {
+			result.push(el);// Push mach
+		}
+	});
+
+	if (result.length >=1) {
+		return result;// Return matching values
+	} else {
+		return null;// No match, return null
+	}
+}
+
+var a = ['a','b','c','d','c'], b = ['a','c','f','h'];
+// Return matching value
+console.log(getMatchingValue(a, b));
```

Comment:
Try to switch to `console.assert` as in:
`console.assert(getMatchingValue(['a','b','c','d','c'], ['a','c','f','h']) === ['a', 'c'])`
Play with console.assert to see what would happen if the test fails.